Asgardia.Constitution.Proposal.txt

```diff
@@ -155,42 +154,40 @@ ASGARDIAN RESOURCES
 Article 11. Human Resources
 § 1. Asgardia encourages the development of human resources.
 § 2. Asgardia uses its own material, financial, and other resources to ensure free access to distance learning, professional qualifications, and scientific and creative work for Asgardian citizens.
-§ 3. Asgardian citizens are entitled to free time for self-development as well as self-improvement, creative and cultural pursuits for a minimum of 25 percent of the year while employed by a business operating within Asgardian territories. It is not required to spend this time contiguously.
-§ 4. Asgardia works to create jobs and facilitate maximum utilisation of the abilities and qualifications of its citizens.
+§ 3. Asgardian citizens are entitled to free time for self-development, self-improvement, and creative and cultural pursuits for a minimum of 35 percent of days in the Earth year while employed by a business operating within Asgardian territories. It is not required to spend this time contiguously.

```

Comment:
This needs to be clarified. At a 40 hour work week, with weekends off, depending on if you’re looking at by hours or by days, you already achieve 25%. If the goal here is to reduce the number of days/hours worked over the standard today, this does not achieve that. I updated this to 35% as this would equate to ~6 weeks of vacation per year, if calculated by number of working days.